test/dlc_tests/container_tests/bin/pytorch_tests/setupPyTorchBackendTest

```diff
@@ -20,15 +29,6 @@ source ~/.bashrc
 # setup benchmark repo
 git clone https://github.com/pytorch/benchmark
 pushd benchmark
-git reset --hard 020aac26
+git reset --hard 78e54615
```

Comment:
How is this commit ID being decided? How should it be modified in the future?

Why does this commit ID need to be hardcoded in the test, instead of being dynamically defined based on something like PyTorch version in the image being tested, or something like that?

Reply:
Yes, I hope to pin this more properly in the future. Since the torch.compile is still new, we've been observing frequent errors if it's not pinned. And unfortunately, https://github.com/pytorch/benchmark doesn't have version tags for now. I've been manually updating the commit to latest every time I work on the backend test.